api/src/game/classes/Player.ts

```
@@ -0,0 +1,30 @@
+export class Player {
+  public socket?: GameSocket;
+  constructor(
+    private user: User,
+    private context: GameContext,
+  ) {}
+  public get id(): string {
+    return this.user.id;
+  }
+
+  isConnected(): this is this & { socket: GameSocket } {
+    return this.socket !== undefined && this.socket.connected;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)